### PR TITLE
[MAINTENANCE] GCP Integration Pipeline fix

### PR DESCRIPTION
**Pull request by Shinnnyshinshin**
### Overview

1. Renaming of `test_expectations_cfe` and `test_expectations` to `v2` and `v3` (follow up to #6190)
2. Adjusting V3 Expectations for `expect_column_values_to_be_between_tz_informed`

    **What was the issue?**

    * Comparison between STRING and DATETIME is not supported by `bigquery`
    ```bash
    E great_expectations.exceptions.exceptions.MetricResolutionError: 
    (google.cloud.bigquery.dbapi.exceptions.DatabaseError) 400 No matching signature for operator >= for 
    argument types: STRING, DATETIME. Supported signature: ANY >= ANY at [3:33]
    ```
3. Adjusting V2 Expectations for `expect_column_values_to_match_like_pattern` and `expect_column_values_to_not_match_like_pattern`. 
    
     **What was the issue?**

    * Like-pattern resolution in `bigquery` results in a `NotImplementedError`
    ```bash
    like_pattern_expression = self._get_dialect_like_pattern_expression(
            column, like_pattern, positive=False
        )
        if like_pattern_expression is None:
            logger.warning(
                f"Like patterns are not supported for dialect {str(self.sql_engine_dialect)}"
            )
     >           raise NotImplementedError
     E           NotImplementedError


    ```

* This PR renames the files and suppresses the tests for `bigquery`

### Definition of Done
- [x] My code follows the Great Expectations [style guide](https://docs.greatexpectations.io/docs/contributing/style_guides/code_style)
- [x] I have performed a [self-review](https://docs.greatexpectations.io/docs/contributing/contributing_checklist) of my own code
- [x] I have run any local integration tests and made sure that nothing is broken.

### CODE REVIEW

File: azure-pipelines-gcp-integration.yml [hunk 1]
```diff
@@ -117,6 +117,3 @@ stages:
               GOOGLE_APPLICATION_CREDENTIALS: $(gcp_authkey.secureFilePath)
               GE_TEST_GCP_PROJECT: $(GE_TEST_GCP_PROJECT)
               GE_TEST_BIGQUERY_DATASET: $(GE_TEST_BIGQUERY_DATASET)
-          # The pip freeze output could be helpful to reproduce performance test results.
-          - publish: pip-freeze.txt
-            artifact: PipFreeze
```
Comment (Shinnnyshinshin) [inline azure-pipelines-gcp-integration.yml:122]: `Pipfreeze` has not been used to reproduce test results, and is now giving errors in our pipeline.

```bash
ApplicationInsightsTelemetrySender correlated 2 events with X-TFS-Session 7df3e132-aada-4a43-b781-0a06fe1923db
##[error]Artifact PipFreeze already exists for build 119157.
Finishing: PublishPipelineArtifact
```

File: azure-pipelines-gcp-integration.yml [hunk 2]
```diff
@@ -74,10 +74,10 @@ stages:
         strategy:
           matrix:
             expectations_cfe:
```
Comment (kenwade4) [inline azure-pipelines-gcp-integration.yml]: May as well rename these labels in the matrix from expectations_cfe to expectations_v3_api and expectations to expectations_v2_api

**Review by kenwade4 — COMMENTED** [review]
Looks good, just a minor change request

File: azure-pipelines-gcp-integration.yml (hunk 2 above)
Comment (Shinnnyshinshin) [reply azure-pipelines-gcp-integration.yml]: great catch @kenwade4 thank you. Added suggestion

**Review by anthonyburdi — APPROVED** [review]
LGTM! Thank you for this fix.

**Pull request merged**